### PR TITLE
Remove old docs, migrate to new upload release asset script, simplify…

### PR DESCRIPTION
**Pull request by rickducott**
… make and cloudbuild

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -114,4 +114,4 @@
 
 [[constraint]]
   name = "github.com/solo-io/go-utils"
-  version = "v0.2.4"
```
Comment (rickducott) [inline Gopkg.toml]: Will upgrade to version when https://github.com/solo-io/go-utils/pull/37 merges

**Comment by ilackarms** [comment]
note that we seem to have a flake in ci:

`Step #5 - "test": Message: "resource quota evaluates timeout",`

with some googling it looks like a low-level kubernetes error related to etcd. it might be worth investigating further as it's not the first time i've seen this flake. however i don't think it's related to our code (most likely an issue with concurrent tests against the GKE cluster)

**Comment by ilackarms** [comment]
see https://console.cloud.google.com/gcr/builds/8f4171af-8cd2-4402-a6f6-578e198418fe?project=907026730415 for details on the flake

**Comment by rickducott** [comment]
Hmm, looks like the cluster is in a bad state: 
```
[11:57:26] rick@Ricks-MacBook-Pro-2:~/code2/solobot/contrib$ kubectl get all -n test64irz
NAME                                         READY   STATUS             RESTARTS   AGE
pod/istio-citadel-757dbf66b4-vp95r           1/1     Running            0          15h
pod/istio-cleanup-secrets-8pkkv              0/1     Completed          1          15h
pod/istio-egressgateway-674686c846-28rmw     1/1     Running            0          15h
pod/istio-galley-6dd767999-nzr5p             1/1     Running            0          15h
pod/istio-ingressgateway-6bbdd58f8c-rhvm5    1/1     Running            0          15h
pod/istio-pilot-7fc859bf4f-85c8p             2/2     Running            4          15h
pod/istio-policy-6d8874997b-b8g6g            1/2     CrashLoopBackOff   11         21m
pod/istio-policy-6d8874997b-k8lth            2/2     Running            6          15h
pod/istio-security-post-install-rwzj5        0/1     Completed          2          15h
pod/istio-sidecar-injector-b88dfb954-tftk5   1/1     Running            0          15h
pod/istio-telemetry-75c6684bc5-dqkp9         1/2     CrashLoopBackOff   11         21m
pod/istio-telemetry-75c6684bc5-fx59t         2/2     Running            6          15h
pod/prometheus-76db5fddd5-7g9q2              1/1     Running            0          15h

NAME                             TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)                                                                                                                   AGE
service/istio-citadel            ClusterIP      10.11.246.139   <none>          8060/TCP,9093/TCP                                                                                                         15h
service/istio-egressgateway      ClusterIP      10.11.249.22    <none>          80/TCP,443/TCP                                                                                                            15h
service/istio-galley             ClusterIP      10.11.243.250   <none>          443/TCP,9093/TCP                                                                                                          15h
service/istio-ingressgateway     LoadBalancer   10.11.241.68    35.188.50.242   80:31380/TCP,443:31390/TCP,31400:31400/TCP,15011:30402/TCP,8060:31879/TCP,853:31238/TCP,15030:32605/TCP,15031:31150/TCP   15h
service/istio-pilot              ClusterIP      10.11.246.246   <none>          15010/TCP,15011/TCP,8080/TCP,9093/TCP                                                                                     15h
service/istio-policy             ClusterIP      10.11.242.161   <none>          9091/TCP,15004/TCP,9093/TCP                                                                                               15h
service/istio-sidecar-injector   ClusterIP      10.11.246.239   <none>          443/TCP                                                                                                                   15h
service/istio-telemetry          ClusterIP      10.11.253.61    <none>          9091/TCP,15004/TCP,9093/TCP,42422/TCP                                                                                     15h
service/prometheus               ClusterIP      10.11.240.216   <none>          9090/TCP                                                                                                                  15h

NAME                                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/istio-citadel            1         1         1            1           15h
deployment.apps/istio-egressgateway      1         1         1            1           15h
deployment.apps/istio-galley             1         1         1            1           15h
deployment.apps/istio-ingressgateway     1         1         1            1           15h
deployment.apps/istio-pilot              1         1         1            1           15h
deployment.apps/istio-policy             2         2         2            1           15h
deployment.apps/istio-sidecar-injector   1         1         1            1           15h
deployment.apps/istio-telemetry          2         2         2            1           15h
deployment.apps/prometheus               1         1         1            1           15h

NAME                                               DESIRED   CURRENT   READY   AGE
replicaset.apps/istio-citadel-757dbf66b4           1         1         1       15h
replicaset.apps/istio-egressgateway-674686c846     1         1         1       15h
replicaset.apps/istio-galley-6dd767999             1         1         1       15h
replicaset.apps/istio-ingressgateway-6bbdd58f8c    1         1         1       15h
replicaset.apps/istio-pilot-7fc859bf4f             1         1         1       15h
replicaset.apps/istio-policy-6d8874997b            2         2         1       15h
replicaset.apps/istio-sidecar-injector-b88dfb954   1         1         1       15h
replicaset.apps/istio-telemetry-75c6684bc5         2         2         1       15h
replicaset.apps/prometheus-76db5fddd5              1         1         1       15h

NAME                                                       REFERENCE                         TARGETS   MINPODS   MAXPODS   REPLICAS   AGE
horizontalpodautoscaler.autoscaling/istio-egressgateway    Deployment/istio-egressgateway    40%/80%   1         5         1          15h
horizontalpodautoscaler.autoscaling/istio-ingressgateway   Deployment/istio-ingressgateway   40%/80%   1         5         1          15h
horizontalpodautoscaler.autoscaling/istio-pilot            Deployment/istio-pilot            8%/80%    1         5         1          15h
horizontalpodautoscaler.autoscaling/istio-policy           Deployment/istio-policy           75%/80%   1         5         2          15h
horizontalpodautoscaler.autoscaling/istio-telemetry        Deployment/istio-telemetry        75%/80%   1         5         2          15h

NAME                                    DESIRED   SUCCESSFUL   AGE
job.batch/istio-cleanup-secrets         1         1            15h
job.batch/istio-security-post-install   1         1            15h
```

**Comment by rickducott** [comment]
deleted all the deployments and got:

```
kubectl get all -n test64irz
error: the server doesn't have a resource type "clusteringresses"
```

**Comment by rickducott** [comment]
There's probably an issue where concurrent builds can leave istio in a bad state, but cleared it out manually for now.

**Pull request merged**